apps/odyssey-stats/src/components/odyssey-query-site-purchases.ts

```diff
@@ -8,47 +8,72 @@ import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
```

Comment:
The whole file should be able to be removed if we ditch support over lower version of Jetpack by releasing a v2 of Odyssey Stats I suppose.